Assets/Package/Runtime/Console/ConsoleGui.cs

```diff
@@ -217,6 +255,26 @@ private void FetchPrevCommand()
             }
         }
 
+        private void PerformAutocomplete()
+        {
+            if (string.IsNullOrEmpty(currentAutocompleteMatch))
+            {
+                return;
+            }
+
+            if (currentInput.Contains(' '))
+            {
+                currentInput = currentInput.Trim().Substring(0, currentInput.LastIndexOf(' '));
```

Comment:
Wydaje mi się, że można to zrobić regexem i wtedy wyjdzie bez tego ifa i substringa.

Reply:
spróbuję

Reply:
to tutaj jeszcze Trim przed ifem inaczej możesz dostać -1 w indeksie

Reply:
chatgpt leży i kwiczy :/

Reply:
Spróbuję wykminić rano

Reply:
tutaj zrób po prostu:
```
Match match = Regex.Match(trimmedInput, @"^(.*)\s[^\s]+$");
currentInput = match.Success 
? ${match.Groups[1].Value} {currentAutocompleteMatch}
: currentAutocompleteMatch;
```